src/Symfony/Component/DependencyInjection/Tests/Loader/FileLoaderTest.php

```diff
@@ -49,15 +50,22 @@
 
 class FileLoaderTest extends TestCase
 {
+    use ExpectUserDeprecationMessageTrait;
+
     protected static string $fixturesPath;
 
     public static function setUpBeforeClass(): void
     {
         self::$fixturesPath = realpath(__DIR__.'/../');
     }
 
+    /**
+     * @group legacy
+     */
     public function testImportWithGlobPattern()
     {
+        $this->expectUserDeprecationMessage('Since symfony/dependency-injection 7.4: XML configuration format is deprecated, use YAML or PHP instead.');
```

Comment:
maybe remove/spli the xml part? a legacy test case is usually planned for removal, which isn't the case for this one, right?

Reply:
I assumed we could simply remove the XML part from the test on 8.0 but after a second look it seems it would be better to replace the XML config with a PHP one in this PR.

Reply:
Done